.ci/pytorch/test.sh

```diff
@@ -521,6 +521,51 @@ test_inductor_torchbench_smoketest_perf() {
   done
 }
 
+test_inductor_torchbench_cpu_smoketest_perf(){
+  TEST_REPORTS_DIR=$(pwd)/test/test-reports
+  mkdir -p "$TEST_REPORTS_DIR"
+
+  #set jemalloc
+  #JEMALLOC_LIB="$(dirname "$(which python)")/../lib/libjemalloc.so"
+  JEMALLOC_LIB="/usr/lib/x86_64-linux-gnu/libjemalloc.so.2"
```

Comment:
How do we make sure libjemalloc is installed?

Reply:
It's installed in base docker build stage, refer https://github.com/pytorch/pytorch/blob/main/.ci/docker/common/install_base.sh#L64